dakota_manylinux_install_files/build_wheels_local.sh

```diff
@@ -0,0 +1,150 @@
+INSTALL_DIR=/tmp/INSTALL_DIR
+
+mkdir -p $INSTALL_DIR
+mkdir -p /tmp/trace
+touch /tmp/trace/boost_bootstrap.log
+touch /tmp/trace/boost_install.log
+touch /tmp/trace/dakota_bootstrap.log
+touch /tmp/trace/dakota_install.log
+touch /tmp/trace/env
+
+# VERY IMPORTANT: extract python dev headers,
+# more info: https://github.com/pypa/manylinux/pull/1250
+pushd /opt/_internal && tar -xJf static-libs-for-embedding-only.tar.xz && popd
+
+echo "pushd /opt/_internal && tar -xJf static-libs-for-embedding-only.tar.xz && popd" >> /tmp/trace/env
+echo "INSTALL_DIR=$INSTALL_DIR" >> /tmp/trace/env
+
+yum install lapack-devel -y
+yum install python3-devel.x86_64 -y
+yum install -y wget
+wget https://boostorg.jfrog.io/artifactory/main/release/1.82.0/source/boost_1_82_0.tar.bz2 --no-check-certificate
+
+python_exec=$(which python3.10)
+
+$python_exec -m venv myvenv
+source ./myvenv/bin/activate
+pip install numpy
+pip install pybind11[global]
+
+PYTHON_DEV_HEADERS_DIR=$(rpm -ql python3-devel.x86_64 | grep '\.h$' | head -n 1 | xargs dirname)
+NUMPY_INCLUDE_PATH=$(find /tmp -type d -path "*site-packages/numpy/core/include") # $(python -c "import numpy; print(numpy.get_include())")
+PYTHON_INCLUDE_PATH=$(python -c "from distutils.sysconfig import get_python_inc; print(get_python_inc())")
+python_root=$(python -c "import sys; print(sys.prefix)")
+python_version=$(python --version | sed -E 's/.*([0-9]+\.[0-9]+)\.([0-9]+).*/\1/')
+python_version_no_dots="$(echo "${python_version//\./}")"
+python_bin_include_lib="    using python : $python_version : $(python -c "from sysconfig import get_paths as gp; g=gp(); print(f\"$python_exec : {g['include']} : {g['stdlib']} ;\")")"
+
+echo "Found dev headers $PYTHON_DEV_HEADERS_DIR"
+echo "Found numpy include path $NUMPY_INCLUDE_PATH"
+echo "Found python include path $PYTHON_INCLUDE_PATH"
+echo "Found python root $python_root"
+
+tar xf boost_1_82_0.tar.bz2
+cd boost_1_82_0
```

Comment:
This should be picked up through a variable or similar.
Also, the current version of boost is 1.83.